### ISSUE TITLE
The paper data cannot be reproduced

### ISSUE THREAD

**Issue by lihui1998** [tdchaitanya/MMTOD#6]
hello，the accuracy of the pre-training model you provided is 57.9, but it is 58.5 in the paper, so we cannot get the results of the paper.thanks.
